src/services/DogApi.js

```diff
@@ -0,0 +1,40 @@
+const myHeaders = new Headers();
+
+myHeaders
+  .append('Authorization', 
+    'Bearer 1544caee-0010-41ae-92cb-28cbee4c0e70');
+
+export const getDogsByBreed = async() => {
+  return await fetch('https://api.thedogapi.com/v1/breeds', {
+    method: 'GET',
+    headers: myHeaders
```

Comment:
Great use of a this. You could refactor into a cool uber fetch but...you app is small and that would proper over engineer it. Great work on this!